lottie/src/main/java/com/airbnb/lottie/LottieAnimationView.java

```diff
@@ -54,8 +56,12 @@
     Strong
   }
 
-  private static final Map<String, LottieComposition> STRONG_REF_CACHE = new HashMap<>();
-  private static final Map<String, WeakReference<LottieComposition>> WEAK_REF_CACHE =
+  private static final SparseArray<LottieComposition> STRONG_RAW_REF_CACHE = new SparseArray<>();
```

Comment:
This should be `RAW_RES_STRONG_REF_CACHE` (strong goes with ref not raw)
Same for the asset cache.